Recrutify/Recrutify.Services/Services/FormEmailService.cs

```diff
@@ -90,5 +119,37 @@ private Generator CreateGenerator(string templatePath)
             var compiler = new HtmlFormatCompiler();
             return compiler.Compile(mailText);
         }
+
+        private StringBuilder CreateInviteFile(DateTime dateTimeInterview, string description)
+        {
+            StringBuilder str = new StringBuilder();
+            str.AppendLine("BEGIN:VCALENDAR");
+
+            str.AppendLine($"PRODID: {Constants.Company.Name}");
+            str.AppendLine("VERSION:2.0");
+            str.AppendLine("METHOD:REQUEST");
+            str.AppendLine("TZOFFSETTO:+0300");
```

Comment:
Move + 3 in constant for all places you use it, what if next requirement will be using African time - it S not comfortable to change time in 10 places in code

Reply:
+